src/translations/index.js

```diff
@@ -22,5 +22,11 @@ export default {
         SEARCH_PLACEHOLDER: 'Arama...',
         SELECTED_ITEMS_COUNT_TEXT: '{count} öğe seçildi',
         NOTHING_TO_SHOW: 'Gösterecek hiçbir şey yok!'
+    },
+    RU: {
+        PLACEHOLDER: 'Выберите элемент',
+        SEARCH_PLACEHOLDER: 'Введите текст...',
+        SELECTED_ITEMS_COUNT_TEXT: 'Выбрано элементов: {count}',
```

Comment:
Could you please make it look like {count} item(s) have been selected?

Reply:
I made the translation in this way, because in Russian there are three states of the ending of numbers, in contrast to English.
In English we have "item" if user selected only one item, and "item**s**" if user selected many items.
In Russian we have "элемент" if user selected only one item, "элемент**а**" if user selected 2,3 or 4 items and "элемент**ов**" if user selected more than four items.
That's why I use sentence like this 'Выбрано элементов: {count}', in that case we don't need to worry about word ending.
Sorry for mistakes, I'm not a native English speaker :). I hope you understand what i mean.

Reply:
😂 Omg! Anyway, thanks.